spec/system/gov_one_info_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'Gov One Info' do
+  context 'with an unauthenticated visitor' do
+    before do
+      visit '/gov-one/info'
+    end
+
+    it 'displays the correct content' do
+      expect(page).to have_css('h1', text: 'How to access this training course')
+      expect(page).to have_css('p', text: 'This service uses GOV.UK One Login which is managed by the Government Digital Service.')
+      expect(page).to have_css('p', text: 'You will be asked to sign in to your account, or create a One Login account, in this service')
+      expect(page).to have_css('a', text: 'Continue to GOV.UK One Login')
+    end
+
+    it 'has the correct login link' do
+      link = find_link('Continue to GOV.UK One Login')
```

Comment:
If this complex link is asserted elsewhere I'd favour a simpler, more readable, spec.